### PR TITLE
Update User Interface

### PR DESCRIPTION
**Pull request by DavidMansolino**
Fix issue #897 

Previews:
  - https://www.cyberbotics.com/doc/guide/the-3d-window?version=update-user-interface#selecting-an-object
  - https://www.cyberbotics.com/doc/guide/the-user-interface?version=update-user-interface#main-toolbar

### CODE REVIEW

File: guide/the-3d-window.md [hunk 1]
```diff
@@ -5,8 +5,7 @@
 A single mouse click allows you to select a solid object.
 The bounding object of a selected solid is represented by white lines.
 These lines turn pink if the solid is colliding with another one and blue when the solid is idle, i.e., it comes to rest and it does not interact with any other active solid.
-Selecting a robot enables the `Show Robot Window` item in the `Tools` menu.
-Double-clicking on a solid object opens the Robot Window and if it was closed, the Scene Tree.
```
Comment (DavidMansolino) [inline guide/the-3d-window.md]: > Double-clicking on a solid object opens [...] if it was closed, the Scene Tree.

Was this even working at some point? It doesn't seems to be the case neither on master or develop.

Comment (stefaniapedrazzi) [reply guide/the-3d-window.md]: Yes, I think that at some point (early Webots 7 or previous vesions) this was working.
In any case this is no longer the case now.

**Comment by DavidMansolino** [comment]
👍

File: guide/the-3d-window.md (hunk 1 above)
Comment (DavidMansolino) [reply guide/the-3d-window.md]: Ok thank you, I will remove it for now and open an issue on Webots repository to discuss if we want to restore this feature or not.

**Comment by stefaniapedrazzi** [comment]
:eyes:

File: guide/the-3d-window.md [hunk 2]
```diff
@@ -2,15 +2,25 @@
...
 
+#### Context Menu
+
+%figure "The context menu allows you to perform various actions depending on the current selected object"
+
+![context_menu.png](images/context_menu.png)
+
+%end
+
+When an object is selected in the 3D Window is is possible to right click on it to open the context menu.
```
Comment (stefaniapedrazzi) [inline guide/the-3d-window.md]: An object doesn't need to be selected to open the context menu.
It is the right-click action that picks the object and select it before opening its context menu.

In the rest of the page we use `3D window`

typo: "is is possible" -> "it is possible"

File: guide/the-3d-window.md [hunk 3]
```diff
@@ -2,15 +2,25 @@
...
+This menu allows you to perform various actions such as: copy the object, delete the object, move the viewpoint to the object and show the robot window (if the object is a [Robot](../reference/robot.md)).
```
Comment (stefaniapedrazzi) [inline guide/the-3d-window.md]: I'm not sure that it is correct to use the colon after "such as".
Here (Other tips section) for example they explicitly suggest to not do it:
https://www.aje.com/en/arc/editing-tip-such-and-including/

File: guide/the-user-interface.md [hunk 4]
```diff
@@ -369,8 +369,7 @@ The remaining menu items bring up various information as indicated, in the form
 
 ### Main Toolbar
 
-The main toolbar contains items for editing the world and the speedometer (see this [subsection](#speedometer-and-virtual-time)), other than shortcuts to items of the `File`, `Simulation` and `View` menus.
-Edit actions always apply on the selected object, that is highlighted both in the 3D window and in the Scene Tree.
+The main toolbar contains items for adding new nodes to the world, the speedometer (see this [subsection](#speedometer-and-virtual-time)) and shortcuts to items of the `File`, `Simulation` and `View` menus.
```
Comment (stefaniapedrazzi) [inline guide/the-user-interface.md]: Note that "items for adding new nodes" is actually only one item.

**Comment by DavidMansolino** [comment]
👍

**Pull request merged**